libcxx/test/std/ranges/range.adaptors/range.stride.view/borrowing.compile.pass.cpp

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// expected-no-diagnostics
```

Comment:
The `// expected-no-diagnostics` tag is for `clang -verify`, which is a way of testing which diagnostics the compiler emits. It should only be relevant for `.verify.cpp` tests, so you should remove it here.

Reply:
Thank you! I believe that I have addressed your comment. Feel free to resolve the conversation if you agree!